contrib/gcppubsub/pkg/dispatcher/dispatcher/controller.go

```diff
@@ -36,11 +39,20 @@ const (
 	// controllerAgentName is the string used by this controller to identify itself when creating
 	// events.
 	controllerAgentName = "gcp-pubsub-channel-dispatcher"
+
+	// exponential backoff constants used to limit the pace at which
```

Comment:
Comments should be full sentences, starting with capital letter and ending with punctuation.

Reply:
Done, thanks!